CHANGELOG.md

```diff
@@ -17,6 +17,13 @@ For older changes see the [archived Singularity change log](https://github.com/a
   directly from Dockerfile, without having to use Docker/Podman but only BuildKit.
   The full buildkit log file is included in the image, for traceability.
   It is also showed on the console, as a progress update while building.
+- Add support for selective mounting of Intel(R) Gaudi accelerators.
+  This feature is enabled via the `--intel-hpu` option and by specifying the
+  `HABANA_VISIBLE_DEVICES` environment variable, which should contain
+  a comma-separated list of device IDs (e.g., `"1,2,3"`). This controls
+  which accelerators are bound into the `/dev` directory in the container.
+  This feature only works when a minimal `/dev` directory is used, either by
```

Comment:
To be clear: by "works" do you mean that without `--contain` all the devices will be imported automatically?

If so I will rewrite this changelog entry a little to make that clear, along with mentioning that the default if `HABANA_VISIBLE_DEVICES` is not set is `all`.